pkg/controller/nodelifecycle/scheduler/taint_manager_test.go

```diff
@@ -306,15 +303,10 @@ func TestUpdatePod(t *testing.T) {
 			podIndexer.Add(item.prevPod)
```

Comment:
After `controller.Run()` (a few lines above) you set `controooler.taintedNodes` - that's potentially racy, given that all the operations on that map inside the controller are guarded with locks, that might cause problems, I'd suggest moving that line before running the controller.

Reply:
Moved up, though it was like this before